app.js

```diff
@@ -0,0 +1,353 @@
+/*global alert ,confirm ,window ,setInterval ,setTimeout ,document ,clearInterval*/
+    //  Get user info
+(function getUserInfo() {
+    'use strict';
+    var url = window.location.href,
+        userName,
+        userAge,
+        greetDate,
+        userNamePlace,
+        getTimeNow;
+
+    //  if game page has no data(string) in url, keep moving.
+    //  Otherwise, declare user's name and age variables.
+    if ("string" === typeof url.split("?")[1]) {
+
+        userName = url.split("?")[1].split("=")[1].split("&")[0];
+        userAge = url.split("?")[1].split("=")[2];
+
+        //  Greeting New Guest
+        greetDate = document.getElementById('greeting-date');
+        userNamePlace = document.getElementById('user-name');
+        getTimeNow = new Date().getHours();
+
+        if (getTimeNow >= 5 && getTimeNow <= 11) {
+
+            greetDate.innerHTML = "Good Morning, ";
+            userNamePlace.innerHTML = userName + "!";
+
+        } else if (getTimeNow >= 12 && getTimeNow <= 18) {
+
+            greetDate.innerHTML = "Good Afternoon, ";
+            userNamePlace.innerHTML = userName + "!";
+
+        } else {
+
+            greetDate.innerHTML = "Good Evening, ";
+            userNamePlace.innerHTML = userName + "!";
+        }
+    }
+}());
+
+
+    //  set myGame object and store variables in it to use in functions
+var myGame = {
+    
+    myBody: document.getElementById('box-body-id'),
+    myBox: document.getElementById('box-id'),
+    playerTime: document.getElementById('playerTime'),
+    setTime: null,
+    endTime: null,
+    myTime: null,
+    flashOpt: document.getElementById('flashOpt')
+};
+    
+    //  Set a random color for the box or circle
+function setRandomColor() {
+    'use strict';
+
+    var letters = '0123456789ABCDEF'.split(''),
+        color = '#',
+        i;
+
+    for (i = 0; i < 6; i++) {
+        color += letters[Math.round(Math.random() * 15)];
+    }
+    return color;
+}
+
+    //  Not Working with "onclick" event in js file
+/*var gameLevelsSelect = {
+    levels: document.getElementById('levels-id').children,
+    easy: document.getElementById('easyOpt'),
+    medium: document.getElementById('mediumOpt'),
+    hard: document.getElementById('hardOpt'),
+    flash: document.getElementById('flashOpt')
+}
+gameLevelsSelect.levels.onclick = function () {
+    if (this[0]) {
+        gameLevelsFunctions.lvEasy();
+    }
+}*/
+    //  set levels functions
+
+function lvEasy() {
```

Comment:
All these `lv*` functions should be grouped into a one function which accepts a `level` parameter and acts upon its value.